packages/eslint-plugin/docs/rules/thenable-in-promise-aggregators.md

```diff
@@ -0,0 +1,48 @@
+---
+description: 'Disallow passing non-Thenable values to promise aggregators.'
+---
+
+> 🛑 This file is source code, not the primary documentation location! 🛑
+>
+> See **https://typescript-eslint.io/rules/thenable-in-promise-aggregators** for documentation.
+
+A "Thenable" value is an object which has a `then` method, such as a Promise.
+The `await` keyword is generally used to retrieve the result of calling a Thenable's `then` method.
+
+When multiple Thenable's are running at the same time, it is sometimes desirable to wait until any one of them resolves (`Promise.race`), all of them resolve or any of them reject (`Promise.all`), or all of them resolve or reject (`Promise.allSettled`).
```

Comment:
[Docs] Nit: I know apostrophes to indicate plurals can sometimes be ok, but I've always found them to be ambiguous. Maybe this is just me 😄 

```suggestion
When multiple Thenables are running at the same time, it is sometimes desirable to wait until any one of them resolves (`Promise.race`), all of them resolve or any of them reject (`Promise.all`), or all of them resolve or reject (`Promise.allSettled`).
```